src/pages/MyLibrary/LibraryBooks/OpenBook/OpenBook.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Page, Document } from 'react-pdf';
+import StarRating from './StarRating';
+import useViewPort from './ViewPort/useViewPort';
+import { pdfjs } from 'react-pdf';
+pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
+
+const OpenBook = props => {
+  const { book, setBook } = props;
+  const [numPages, setNumPages] = useState(null);
+  const [pageNumber, setPageNumber] = useState(1);
+  const { width, height } = useViewPort();
+
+  function onDocumentLoadSuccess({ numPages }) {
+    setNumPages(numPages);
+  }
+
+  const closeBook = () => {
+    setBook(book => !book);
+  };
+
+  const bookscale = () => {
+    let newwidth = width / 1920;
+    let newheight = height / 1019;
+
+    return newwidth >= newheight ? newheight : newwidth;
+  };
+
+  return (
+    <>
+      <Modal book={book} onClick={closeBook} />
+      <OpenBookTotal book={book}>
+        <DocumentBox>
+          <Document
+            file="PlantSiteManagementImprovementMethod.pdf"
+            onLoadSuccess={onDocumentLoadSuccess}
+          >
+            <Page pageNumber={pageNumber} scale={bookscale()} />
+          </Document>
+          <MoveButton
+            location="left"
+            onClick={() =>
+              pageNumber > 1 ? setPageNumber(pageNumber - 1) : null
+            }
+          />
+          <MoveButton
+            location="right"
+            onClick={() =>
+              pageNumber < numPages ? setPageNumber(pageNumber + 1) : null
+            }
+          />
+        </DocumentBox>
+        <InfoBox>
+          <Pset>
+            <span>
+              Page {pageNumber} of {numPages}
+            </span>
+            <StarRating />
+          </Pset>
+        </InfoBox>
+      </OpenBookTotal>
+    </>
+  );
+};
+
+const Modal = styled.div`
+  display: ${props => !props.book && 'none'};
+  position: fixed;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.8);
+  z-index: 1;
+`;
+
+const OpenBookTotal = styled.div`
+  display: ${props => (!props.book ? 'none' : 'flex')};
+  justify-content: space-between;
```

Comment:
```suggestion
const Modal = styled.div`
  position: fixed;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: rgba(0, 0, 0, 0.8);
  z-index: 1;
`;

const OpenBookTotal = styled.div`
  display: flex
  justify-content: space-between;
```
위 리뷰에 따라 props로 받아서 사용하는 부분들이 사라질 수 있습니다!